src/constants/actionTypes.js

```diff
@@ -0,0 +1,14 @@
+export const AUTH = {
+  login: "auth/login",
+  logout: "auth/logout",
+};
+
+export const USER = {
+  set: "user/set",
+};
+
+export const HOTEL = {
+  request: "hotel/request",
+  success: "hotel/success",
+  error: "hotel/error",
+};
```

Comment:
I'd like to suggest that you name the entity HOTELS, because the entity represents an array of objects, that is, several hotels
```suggestion
export const HOTELS = {
  request: "hotels/request",
  success: "hotels/success",
  error: "hotels/error",
};
```